syntax_and_semantics/case.md

```diff
@@ -25,7 +25,9 @@ else
 end
 ```
 
-Note that `===` is used for comparing an expression against a `case`'s value.
+For comparing an expression against a `cause`'s value the *case equality operator* `===`. It is defined as a method on [`Object#===`](https://crystal-lang.org/api/Object.html#%3D%3D%3D%28other%29-instance-method) and can be overriden by subclasses to provide meaningful semantics in case statements.
+
+So the exact meaning of the `===` operator is defined by each type. For example, [`Class#===`](https://crystal-lang.org/api/Class.html#%3D%3D%3D%28other%29-instance-method) means an object is an instance of that class, [`Regex#===`](https://crystal-lang.org/api/Regex.html#%3D%3D%3D%28other%3AString%29-instance-method) that a string matches and [`Range#===`](https://crystal-lang.org/api/Range.html#%3D%3D%3D%28value%29-instance-method) that a number is included in that range.
```

Comment:
I rewrote this section as 1 paragraph which flows nicer.

>For comparing an expression against a `case`'s value the *case equality operator* `===` is used. It is defined as a method on [`Object`](https://crystal-lang.org/api/Object.html#%3D%3D%3D%28other%29-instance-method) and can be overriden by subclasses to provide meaningful semantics in case statements. For example, [`Class`](https://crystal-lang.org/api/Class.html#%3D%3D%3D%28other%29-instance-method) defines case equality as when an object is an instance of that class, [`Regex`](https://crystal-lang.org/api/Regex.html#%3D%3D%3D%28other%3AString%29-instance-method) as when the value matches the regular expression and [`Range`](https://crystal-lang.org/api/Range.html#%3D%3D%3D%28value%29-instance-method) as when the value is included in that range.